test/binding.cxx.in

```diff
@@ -0,0 +1,48 @@
+#include <vector>
+
+#include <${mechanism}.hxx>
+
+#include <pybind11/pybind11.h>
+#include <pybind11/stl_bind.h>
+#include <pybind11/stl.h>
+
+typedef std::vector<double> container_t;
```

Comment:
What if we'd like to work with say, [`arrays`](https://en.cppreference.com/w/cpp/container/array)?

Reply:
Leaving this open as I am eager to read your thoughts on this.

Reply:
Yes, I think this is beyond the scope of this PR.